lms/djangoapps/certificates/models.py

```diff
@@ -568,6 +569,20 @@ def handle_course_cert_awarded(sender, user, course_key, **kwargs):  # pylint: d
         fulfill_course_milestone(course_key, user)
 
 
+@receiver(COURSE_CERT_AWARDED, sender=GeneratedCertificate)
+def handle_course_cert_awarded_for_all_prerequisites_milestones(sender, user, course_key, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Shouldn't this signal be captured outside the platform (Pearson-Core)?. @anfbermudezme

Reply:
Now the signal is captured in Pearson-Core.